### PR TITLE
Updates the zero states for home, messaging and favs

### PR DESCRIPTION
**Pull request by orta**
screen shot 2017-11-18 at 1 07 51 pm
screen shot 2017-11-18 at 1 07 42 pm
screen shot 2017-11-18 at 1 07 37 pm

Fixes https://github.com/artsy/collector-experience/issues/719

P.s: @mennenia this doesn't change _much_ in the Home, but it touches it - so you might have a little work on the rebase (mainly I added `.`s, changed a bit of text, changed the title font and turned the BG white ) [see slack](https://artsy.slack.com/archives/C02BAQ5K7/p1511020522000085)

Skip New Tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Components/States/__tests__/ZeroState-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Components/States/__tests__/ZeroState-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by alloy** [comment]
Looks tight 👌

File: src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx [hunk 1]
```diff
@@ -52,22 +51,22 @@ const Icon = styled(Image)`
 export default () =>
   <CenteredView>
     <VerticalLayout>
-      <Title>Buying Art on Artsy is Simple </Title>
+      <Title>BUYING ART ON ARTSY IS SIMPLE</Title>
       <HorizontalLayout>
         <Icon source={require("../../../../../images/find.png")} />
-        <SmallHeadline>Follow artists and find works you love</SmallHeadline>
+        <SmallHeadline>Follow artists & categories and find works you love.</SmallHeadline>
```
Comment (alloy) [inline src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx]: This mixture of ampersand and “and” is the one thing I like less. Can this be a list and use of the Oxford comma?

**Comment by alloy** [comment]
My one comment is not a blocker by any means, feel free to merge.

File: src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx (hunk 1 above)
Comment (orta) [reply src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx]: Think `Follow artists and categories to find works you love` says the same thing, and is simpler - /cc @katarinabatina  ( I took the words from your white BG sketch doc )

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Missing Test Files:

- `src/lib/Scenes/Favorites/Components/Artists/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Artworks/__tests__/index-tests.tsx`
- `src/lib/Scenes/Favorites/Components/Categories/__tests__/index-tests.tsx`
- `src/lib/Components/States/__tests__/ZeroState-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx (hunk 1 above)
Comment (alloy) [reply src/lib/Components/Inbox/Conversations/ZeroStateInbox.tsx]: Good call 👍

**Pull request merged**